app/components/Form/ProjectForm.tsx

```diff
@@ -123,6 +123,14 @@ const ProjectForm: React.FC<Props> = (props) => {
           projectRevisionByProjectRevisionId {
             rank
           }
+          asProject {
```

Comment:
The console has optimisticResponse warnings when editing the project overview. I think these fragment updates need to be propagated to the project revision.